packages/manager/src/features/Linodes/HighPerformanceVolumeIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import BoltIcon from '@mui/icons-material/Bolt';
+import { IconButton, Tooltip } from '@mui/material';
+import React from 'react';
+
+import type { LinodeCapabilities } from '@linode/api-v4';
+
+interface Props {
+  linodeCapabilities: LinodeCapabilities[];
+}
+
+export function HighPerformanceVolumeIcon({ linodeCapabilities }: Props) {
+  const isHighPerformanceVolume = !!linodeCapabilities?.includes(
+    'Block Storage Performance B1'
+  );
+
+  if (!isHighPerformanceVolume) {
+    return null;
+  }
+
+  return (
+    <Tooltip arrow title="High Performance">
```

Comment:
```suggestion
    <Tooltip title="High Performance">
```
To remain consistent with the rest of Cloud Manager's tooltips, we should remove the `arrow` prop here.

Reply:
![Screenshot 2024-12-13 at 12 16 40 PM](https://github.com/user-attachments/assets/dfe901dd-0fb7-44c8-b865-a8e814998ed7)